mkdocs.yml

```diff
@@ -86,12 +86,44 @@ nav:
         - './microservices/system-management/executor/Ch_SysMgmtExecutor.md'
       - Devices Services:
         - './microservices/device/Ch-DeviceServices.md'
-        - './microservices/device/Ch-DeviceServiceList.md'
         - './microservices/device/profile/Ch-DeviceProfile.md'
         - './microservices/device/sdk/Ch-DeviceSDK.md'
         - Device SDK API:
           -  './microservices/device/sdk/SDK-Go-API.md'
-        - './microservices/device/virtual/Ch-VirtualDevice.md'
+        - Supported Device Services:
+          - './microservices/device/Ch-DeviceServiceList.md'
```

Comment:
I don't see this file in list of changes. Expected to see links added for Onvif and Virtual to their local folders

Reply:
Sorry I just got that in sync with the other pr.